### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stevefaulkner**
merge dev into main branch

### CODE REVIEW

File: package/lang/Default.ini [hunk 1]
```diff
@@ -56,7 +56,7 @@ en_mail=sfaulkner@paciellogroup.com
 en_website=https://www.paciellogroup.com/resources/contrastanalyser/
 WATC=http://www.wat-c.org/
 abouttext=The Colour Contrast Analyser was developed by Jun in collaboration with Steve Faulkner.
-versiontext=Colour Contrast Analyser version 2.4
+versiontext=Colour Contrast Analyser version:
```
Comment (patrickhlauke) [inline package/lang/Default.ini:59]: seems that this leaves it dangling as just "Colour Contrast Analyser version:" ?

File: package/lang/Default.ini [hunk 2]
```diff
@@ -130,7 +130,7 @@ res_large_text_AA=Large text %s at Level AA
 res_large_text_AAA=Large text %s at Level AAA
 
 lcr_ratio_is=The contrast ratio is: %s
-contrast_ratio=Contrast ratio: %s
+just_below=The contrast ratio is: just below %s
```
Comment (patrickhlauke) [inline package/lang/Default.ini:133]: this will presumably need to be added (in translated form) to all other language versions?

Comment (ghost) [reply package/lang/Default.ini:133]: Yes.

File: package/lang/Default.ini (hunk 1 above)
Comment (ghost) [reply package/lang/Default.ini:59]: In the latest version, the version number is acquired from the binary file.
ccass

Comment (patrickhlauke) [reply package/lang/Default.ini:59]: Oh, excellent (need to make sure I check that before making a release, so I match the exact version from the code)

File: package/lang/Default.ini (hunk 2 above)
Comment (patrickhlauke) [reply package/lang/Default.ini:133]: Ok I'll see if I can get some kindly volunteers to help out with the remaining ones...

**Comment by patrickhlauke** [comment]
Closing this pull request, going to make a fresh one which includes the above changes to ini files

**Pull request closed without merging**